tutorials/notebooks/coordinates/coordinates.ipynb

```diff
@@ -30,6 +30,7 @@
     "from astropy import units as u\n",
     "from astropy.coordinates import SkyCoord\n",
     "from astropy.table import Table\n",
+    "from astropy.utils import data\n",
```

Comment:
The changes in this file also seem unrelated to this PR.

Reply:
Similar to previous comment, I was trying to get Circle CI and Travis CI passed, since some errors related to this file was thrown after I fixed the broken logo link.